README.md

```diff
@@ -4,7 +4,9 @@
 
 This contract contains a collection of deployments of the contract of the [Safe contracts repository](https://github.com/safe-global/safe-contracts). 
 
-For each deployment the address on the different networks and the abi files are available. To get an overview of the available versions check the available [json assets](./src/assets/).
+For each deployment the address on the different networks and the abi files are available. To get an overview of the available versions check the available [json assets](./src/assets/) and [Summary Table](./SUMMARY.md).
+
+To regenerate the summary table to include new deployments run: `ts-node src/generateSummary.ts`.
```

Comment:
```suggestion
To regenerate the summary table to include new deployments run: `npx ts-node src/generateSummary.ts`.
```

This can be run with npx and removing the need for global installation of `ts-node`

Reply:
Tried `npx` but it wasn't generating the table. I've added it as an option.